### DISCUSSION TITLE
GraphQL projectsV2 cursor not working

### DISCUSSION THREAD

**Discussion by daniel-mode** [community/community#49945] [Apps, API and Webhooks]
### Select Topic Area

Bug

### Body

Hello all,
I believe I found a bug.
I'm calling the GraphQL API with the below query to fetch all the projectsV2 objects, but when I add the `cursor` to it, no projectsV2 edges are returned. 
```
query {
      organization(login: "Some Org") { 
          projectsV2(first: 100) {
              edges {
                  node {
                      id                   
                  }
                  cursor
              }
              totalCount
          }
      }
  }
```
If I remove the cursor, the query returns the first 100 projects.

**Comment by divyamsavsaviya** [comment]
Hi @daniel-mode ,

If you're having trouble fetching all the projectsV2 objects under the "Some org" organisation, you can try using the following GraphQL query:

```
query {
    organization(login: "Some org") {
        projectsV2(first: <perPage>, after: "<endCursor>") {
            nodes {
                id
            }
            pageInfo {
                hasNextPage
                endCursor
            }
        }
    }
}
```

Make sure to replace **<perPage>** with the number of objects you want to fetch per page, and **<endCursor>** with the cursor of the last object in the previous page (if any). For the first page, the value of endCursor would be **null**. This query should return all the projectsV2 objects for the specified organisation by paginating through the results.
